lib/pages/event_creation/event_creation.dart

```diff
@@ -78,7 +87,17 @@ class _EventCreationPageState extends State<EventCreationPage> {
   }
 
   void finishEventCreation() {
-    databaseReference.update(toJson());
+    Event event = Event(
+      hostId: "TEMP VALUE CHANGE LATER",
```

Comment:
imo uzycie null byloby czystsze

Reply:
A może być null w firestore? Nie byłem pewny tbh. Ogólnie to potem hostId nigdy nie będzie null jak już będziemy zapisywać kto zrobił event.